### PR TITLE
feat(ui): add simple Dialog component [LW-7032]

### PR DESCRIPTION
**Pull request by przemyslaw-wlodek**
# Checklist

- [x] JIRA - LW-7032
- [x] Screenshots added.

---

## Proposed solution
New Dialog component, that uses `@radix-ui/react-alert-dialog` primitives and brings simplified API for the end-user.

## Screenshots
image
image

---
# Allure report
`allure-report-publisher` generated test report!

**smokeTests**: ❌ [test report](https://lace-qa:8PP5wBaDV6UoXj@dq4ajm5i5q7bz.cloudfront.net/linux/chrome/2174/6278282540/index.html) for [19894a08](https://github.com/input-output-hk/lace/pull/519/commits/19894a08ccc828a7ce0472717751847c0e58f080)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 31     | 1      | 0       | 0     | 32    | ❌     |

### CODE REVIEW

File: packages/ui/package.json [hunk 1]
```diff
@@ -17,19 +17,20 @@
   "typings": "dist/index.d.ts",
   "scripts": {
     "build": "run -T rollup -c rollup.config.js",
-    "build-storybook": "export NODE_OPTIONS=--openssl-legacy-provider; build-storybook",
```
Comment (greatertomi) [inline packages/ui/package.json:20]: Why are you changing this and by extension the other changes in this file? 🤔

**Review by greatertomi — COMMENTED** [review]
Amazing work. Well done. 💪🏽

File: packages/ui/package.json (hunk 1 above)
Comment (przemyslaw-wlodek) [reply packages/ui/package.json:20]: I did it as a "scout duty" to make it clean (it's a single commit). `export` keyword is redundant.

File: packages/ui/src/design-system/dialog/dialog-actions.css.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { sx } from '../../design-tokens';
+
+export const dialogActions = sx({
+  display: 'flex',
+  justifyContent: 'center',
+  gap: '$16',
+  // TODO waiting for design team decision
```
[the remark this answers is not in the document]
Comment (przemyslaw-wlodek) [reply packages/ui/src/design-system/dialog/dialog-actions.css.ts:7]: Yes, design review is the next step and I'm gonna ask them. :) But thanks for commenting on this

**Review by mrcnk — APPROVED** [review]

**Pull request merged**